Foundation/URLSession/URLSessionTask.swift

```diff
@@ -57,6 +57,8 @@ open class URLSessionTask : NSObject, NSCopying {
     internal convenience init(session: URLSession, request: URLRequest, taskIdentifier: Int) {
         if let bodyData = request.httpBody {
             self.init(session: session, request: request, taskIdentifier: taskIdentifier, body: _Body.data(createDispatchData(bodyData)))
+        }else if let bodyStream = request.httpBodyStream {
```

Comment:
Minor Nit: missing space between leading } and else

Reply:
@iainsmith oh I miss it, thanks!